openapi/components/requestBodies/pastoral-item.yaml

```diff
@@ -0,0 +1,129 @@
+            Required only if Allow specification of severity on pastoral records config is ON.
+          type: integer
+        body_public:
+          description: |
+            General content of this pastoral record in raw HTML
+          type: string
+        body_private:
+          description: |
+            Confidential content of this pastoral record in raw HTML
+          type: string
+        public_attachments[]:
+          nullable: true
+          type: array
+          description: |
+            Files to attach to general content of the pastoral record, supplied as pointers to a file in Schoolbox.
+            Each pointer will be in the format:
+            `$locator:$location:$filename`
+            The variables within the pointer have this meaning:
+            * `$locator` represents the strategy for finding the file:
+              * `store` for files uploaded to Schoolbox's storage
+              * `browse` finds files in Schoolbox's resource browser
+              * `myfile` finds files in a Schoolbox user's My Files browser
+              * `office365` finds files in an Office365 store
+            * `$location` is the location of the file within `$locator`:
+              * for `store` this is a file's unique hash
+              * for `browse` and `myfile` this is a file ID
+             * for `office365` this is the file's unique node
+            * `$filename` is the filename that the file will be given once it is uploaded to wherever it is being uploaded
+            See the [Upload a file to Schoolbox](#operation/storageAsyncUpload) endpoint for details on how to upload a file, and receive a file pointer from that upload.
+        private_attachments[]:
+          nullable: true
+          type: array
+          description: |
+            Files to attach to confidential content of the pastoral record, supplied as pointers to a file in Schoolbox.
+            Each pointer will be in the format:
+            `$locator:$location:$filename`
+            The variables within the pointer have this meaning:
+            * `$locator` represents the strategy for finding the file:
+              * `store` for files uploaded to Schoolbox's storage
+              * `browse` finds files in Schoolbox's resource browser
+              * `myfile` finds files in a Schoolbox user's My Files browser
+              * `office365` finds files in an Office365 store
+            * `$location` is the location of the file within `$locator`:
+              * for `store` this is a file's unique hash
+              * for `browse` and `myfile` this is a file ID
+             * for `office365` this is the file's unique node
+            * `$filename` is the filename that the file will be given once it is uploaded to wherever it is being uploaded
+            See the [Upload a file to Schoolbox](#operation/storageAsyncUpload) endpoint for details on how to upload a file, and receive a file pointer from that upload.
```

Comment:
there is already a schema for attaching files, should we be using just referencing that?
`openapi/components/schemas/filePointer.yaml` ??
It looks like it is being used in newsItem.yaml
```
items:
    $ref: ../schemas/filePointer.yaml
```

Reply:
I tried it first and couldn't make it displayed in the array object.... Does it work for you?

Reply:
same, just tried, it looks like in news it is also specifying the description, and using `$ref` ... weird
lets just leave it here as it. other than updating the link `Upload a file to Schoolbox`

Reply:
Actually what 
```
          items:
            $ref: ../schemas/filePointer.yaml
```
in openapi/components/requestBodies/newsItem.yaml is for?

Reply:
it looks like it doesn't really do anything....
![image](https://user-images.githubusercontent.com/4447143/202962778-c5cae954-e56d-472c-85c6-4ce94dbd9a16.png)

https://github.com/alaress/schoolbox-api-docs/pull/14/commits/4270fa6782949c60104c7a1d6951f0010399472f